src/cheri/attributes.adoc

```diff
@@ -67,8 +67,18 @@ endif::support_varxlen[]
 :rvy_zicbop_mod_file_name: {cheri_base_ext_name}_Zicbop_mod_insns_table_body
 // 32-bit encodings added by Zyhybrid
 :rvy_hybrid_file_name:     {cheri_default_ext_name}_insns_table_body
-// default encoding scheme
-:rvy_default_encoding_name: Zydefaultcap
+// university (default) encoding scheme
+// For some reason can't use _Lymw{cap_rv32_mw_width}e{cap_rv32_exp_max}rc1pc
+// here, probably need some macro for double expansion
+:rvy32_uni_encoding_suffix: _Lymw10e24rc1pc
```

Comment:
and an "h" for "hybrid supported"?

Reply:
No this should not be needed, we are just assigning meaning to reserved bits in hybrid mode for M.